website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -0,0 +1,346 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_backup_schedule"
+sidebar_current: "docs-mongodbatlas-resource-cloud-backup-schedule"
+description: |-
+    Provides a Cloud Backup Schedule resource.
+---
+
+# mongodbatlas_cloud_backup_schedule
+
+`mongodbatlas_cloud_backup_schedule` provides a cloud backup schedule resource. The resource lets you create, read, update and delete policies items.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage - Create a Cluster with no policies
+
+```hcl
+resource "mongodbatlas_cluster" "my_cluster" {
+  project_id   = "<PROJECT-ID>"
+  name         = "clusterTest"
+  disk_size_gb = 5
+
+  //Provider Settings "block"
+  provider_name               = "AWS"
+  provider_region_name        = "EU_CENTRAL_1"
+  provider_instance_size_name = "M10"
+  cloud_backup     = true // must be enabled in order to use cloud_backup_schedule resource
+  provider_disk_iops          = 100
+}
+
+resource "mongodbatlas_cloud_backup_schedule" "test" {
+  project_id   = mongodbatlas_cluster.my_cluster.project_id
+  cluster_name = mongodbatlas_cluster.my_cluster.name
+
+  reference_hour_of_day    = 3
+  reference_minute_of_hour = 45
+  restore_window_days      = 4
+
+}
+```
+
+## Example Usage - Create a Cluster with no policies and add 4 policies items
```

Comment:
Change line 42 to: 
 ## Example Usage - Add 4 Policies Items To A Cluster With Cloud Backup Previously Enabled but with No Policy Items. 

Then add a blank line and the following:
If you followed the example to Create a Cluster with Cloud Backup Enabled but No Policy Items and then want to add policy items later to the mongodbatlas_cloud_backup_schedule this example shows how.